.github/actions/javascript/checkAndroidStatus/checkAndroidStatus.ts

```diff
@@ -0,0 +1,98 @@
+import * as core from '@actions/core';
+import {google} from 'googleapis';
+import CONST from '@github/libs/CONST';
+import GithubUtils from '@github/libs/GithubUtils';
+
+const PACKAGE_NAME = core.getInput('PACKAGE_NAME', {required: true});
+const GOOGLE_KEY_FILE = core.getInput('GOOGLE_KEY_FILE', {required: true});
+const HALTED_STATUS = 'halted';
+
+async function checkAndroidStatus() {
+    const auth = new google.auth.GoogleAuth({
+        keyFile: GOOGLE_KEY_FILE,
+        scopes: ['https://www.googleapis.com/auth/androidpublisher'],
+    });
+
+    const androidApi = google.androidpublisher({
+        version: 'v3',
+        auth,
+    });
+
+    try {
+        // Insert an edit to get an edit ID
+        const editResponse = await androidApi.edits.insert({
+            packageName: PACKAGE_NAME,
+        });
+        const editId = editResponse.data.id ?? 'undefined';
+
+        // Get the production track status
+        const trackResponse = await androidApi.edits.tracks.get({
+            packageName: PACKAGE_NAME,
+            editId,
+            track: 'production',
+        });
+
+        const status = trackResponse.data.releases?.[0]?.status ?? 'undefined';
+        console.log('Track status:', status);
+
+        // Check if the status is halted
+        const HALTED = status === HALTED_STATUS;
+        core.setOutput('HALTED', HALTED);
```

Comment:
Does `setOutput` return from the current action flow?

Reply:
Yes, correct